cmd/relay/bgs/validator.go

```diff
@@ -129,36 +132,56 @@ func (val *Validator) VerifyCommitMessage(ctx context.Context, host *models.PDS,
 	hostname := host.Host
 	hasWarning := false
 	commitVerifyStarts.Inc()
-	logger := slog.Default().With("did", msg.Repo, "rev", msg.Rev, "seq", msg.Seq, "time", msg.Time)
+	logger := slog.Default().With("host", hostname, "did", msg.Repo, "rev", msg.Rev, "seq", msg.Seq, "time", msg.Time)
 
 	did, err := syntax.ParseDID(msg.Repo)
 	if err != nil {
 		commitVerifyErrors.WithLabelValues(hostname, "did").Inc()
-		return nil, err
+		if !val.Sync11ErrorsAreWarnings {
```

Comment:
I think this one (invalid DID syntax) should probably always be an error (eg, not a sync v1.1 change)